deps/postject/src/dist/postject-api.h

```diff
@@ -0,0 +1,181 @@
+// Copyright (c) 2022 Postman, Inc.
```

Comment:
What's the license for this? I think the License should be included

Reply:
@mcollina It is MIT. In any case, we should probably remove this Postman copyright thingy @RaisinTen @dsanders11

Reply:
I have added the license and removed the postman copyright